script/docker.sh

```diff
@@ -27,10 +27,10 @@ bundle config
 echo
 
 echo
-echo "======= NPM ======="
+echo "======= Node ======="
```

Comment:
Not a big deal but I think we should be explicit and change NPM for Yarn.

Reply:
In the end we are managing Node packages.

Reply:
To be completely precise it's npm packages but again, not a big deal 😝

Reply:
npm stands for Node Package Manager ;)

Reply:
Is this file still required??? 😱